sprokit/src/sprokit/pipeline/CMakeLists.txt

```diff
@@ -110,9 +110,7 @@ target_link_libraries(sprokit_pipeline
                       vital_logger
                       ${Boost_CHRONO_LIBRARY}
                       ${Boost_SYSTEM_LIBRARY}
-  PRIVATE             ${Boost_FILESYSTEM_LIBRARY}
-                      ${Boost_DATE_TIME_LIBRARY}
-                      ${Boost_THREAD_LIBRARY}
+  PRIVATE             ${Boost_THREAD_LIBRARY}
```

Comment:
This is not really a comment for this branch, but since I see it, what do we still need Boost Thread for?  I was under the impression that std::thread is a complete replacement for boost::thread.